### PR TITLE
fix issue with rest request step because User needs to be APIAccessor…

### PR DESCRIPTION
**Pull request by pruan-rht**
… instead of User ojbect

### CODE REVIEW

**Comment by pruan-rht** [comment]
this applies to `v3` branch

File: features/step_definitions/common.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@
 end
 
 When /^I perform the :([a-z_]*?) rest request$/ do |yaml_key|
-  @result = env.rest_request_executor.exec(user: user, req: yaml_key.to_sym)
+  @result = user.rest_request(yaml_key.to_sym)
```
Comment (akostadinov) [inline features/step_definitions/common.rb]: good

Comment (akostadinov) [reply features/step_definitions/common.rb]: sehr gut

Comment (pruan-rht) [reply features/step_definitions/common.rb]: @akostadinov...had to do Google Translate this...now I know a German phrase. :)

**Pull request merged**